src/js/containers/homepage/CovidHighlights.jsx

```diff
@@ -284,19 +283,15 @@ export class CovidHighlights extends React.Component {
                 <div id="covid-hero__wrapper" className="covid-hero__wrapper">
                     <div className="covid-hero__content">
                         <h1 className="covid-hero__headline" tabIndex={-1}>
-                            <span>As of {isFetchLatestFyLoading ? <div className="dot-pulse" /> : `${latestSubmissionDate.format('MMMM DD[,] YYYY')},`}</span>
-                            <span>the Federal Government has spent </span>
                             <span>
+                                The federal government has spent
                                 {isAmountLoading && <div className="dot-pulse" />}
```

Comment:
I know this isn't your change, but this class is an animation on an empty div. What should be animated? In checking, I had a long delay before the api returned the totalSpendingAmount, and it was just a blank area with no indication of activity.

Reply:
Thanks for catching this! This is because we changed the background color to a lighter color similar to the animation. Will check in with design today to determine the animation color